README.md

```diff
@@ -14,9 +14,9 @@
 
 # 📗 Table of Contents
 
-- [mymathmagiciansapp](#mymathmagiciansapp)
+- [math-magicians](#math-magicians)
 - [📗 Table of Contents](#-table-of-contents)
-- [📖 \[mymathmagiciansapp\] ](#-mymathmagiciansapp-)
+- [📖 \[math-magicians\] ](#-math-magicians-)
   - [🛠 Built With ](#-built-with-)
     - [Tech Stack ](#tech-stack-)
     - [Key Features ](#key-features-)
```

Comment:
- [ ] Kindly fix the live demo link in your readme header. It is broken. You can link it to a section or remove it.

Reply:
I have done the changes you suggested. Thanks a lot for your kind feedback.